cmd/image-builder/README.md

```diff
@@ -1,21 +1,21 @@
 # Image Builder
 
 Image Builder is a tool for building OCI-compliant images in an SLC-29-compliant system from a GitHub workflow.
-It signs images with a signify service to verify that the image comes from a trusted repository and has not been altered in the meantime.
-It pushes images to Google Cloud Artifact Registry.
+It signs images with a Signify service and pushes them to Google Cloud Artifact Registry.
```

Comment:
"a Signify service" or "the Signify service"?

Reply:
The, of course :) Thanks for spotting this.